package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "async": "2.1.4",
     "bson": "~1.0.4",
+    "clone": "^2.1.1",
```

Comment:
Not going to add an external dependency, especially one with a very dangerous upcarat. Use `require('../../util').clone()` instead please in :point_up: